aviatrix/data_source_aviatrix_vpc.go

```diff
@@ -189,6 +206,55 @@ func dataSourceAviatrixVpcRead(d *schema.ResourceData, meta interface{}) error {
 		log.Printf("[WARN] Error setting private subnets for (%s): %s", d.Id(), err)
 	}
 
+	if vC.CloudType == goaviatrix.AWS {
+		vpc.PublicRoutesOnly = true
+		publicRtbs, err := client.GetVpcRouteTableIDs(vpc)
+		if err != nil {
+			return fmt.Errorf("could not get public vpc route table ids: %v", err)
+		}
+
+		vpc.PublicRoutesOnly = false
+		allRtbs, err := client.GetVpcRouteTableIDs(vpc)
+		if err != nil {
+			return fmt.Errorf("could not get all vpc route table ids: %v", err)
+		}
+
```

Comment:
Default is to output all route tables. So, move getting public route tables to a function like "getPublicRouteTables" similar to "getPrivateRouteTables" below.

Reply:
Good idea. I ended up putting all 3 into functions since it made more sense to me that way